src/java.base/share/classes/sun/security/rsa/RSASignature.java

```diff
@@ -47,7 +47,7 @@
  * @since   1.5
  * @author  Andreas Sterbenz
  */
-public abstract class RSASignature extends SignatureSpi {
```

Comment:
We can probably move the `RSASignature.encodeSignature` method to `RSAUtil` and this class can be package private.